### ISSUE TITLE
How does the learning parameter of a Neural Network can be set

### ISSUE THREAD

**Issue by leondavi** [Artelnics/opennn#201]
Searching the tutorial and API I couldn't find how to control the learning rate parameter. 
If you might please clarify how does the user can control it. 

Thank you.

**Comment by leondavi** [comment]
The learning parameter is a part of optimizer settings: 
https://www.opennn.net/documentation/reference/class_open_n_n_1_1_adaptive_moment_estimation.html 
set_initial_learning_rate (const type &)
